torch/_torch_docs.py

```diff
@@ -2293,14 +2293,18 @@ def merge_dicts(*dicts):
     :func:`torch.stack` concatenates the given sequence along a new dimension.
 
 Args:
-    tensors (sequence of Tensors): any python sequence of tensors of the same type.
-        Non-empty tensors provided must have the same shape, except in the
-        cat dimension.
+    tensors (sequence of Tensors): Non-empty tensors provided must have the same shape,
+        except in the cat dimension.
+
     dim (int, optional): the dimension over which the tensors are concatenated
 
 Keyword args:
     {out}
 
+.. note:: It is recommended to keep all `tensors` in the same dtype to avoid unintended type promotion.
+          For example, mixing `torch.int32` and `torch.int64` may result in the output being automatically
+          promoted to `torch.int64`.
```

Comment:
I'm not sure this is needed. All functions in PyTorch have this behavior of doing type promotion.
If you think it is necessary to leave a comment here, you can make it one sentence and make it not a "may" but "will"

Reply:
If it's a common sense, I think unnecessary to declare this behavior in the method, removed!
Thanks!